runtime/vm/jvminit.c

```diff
@@ -1766,7 +1788,24 @@ IDATA VMInitStages(J9JavaVM *vm, IDATA stage, void* reserved) {
 					}
 					piConfig->sharedClassReadWriteBytes = -1;					/* -1 == proportion of cache size */
 					vm->sharedClassPreinitConfig = piConfig;
+
+					if (argIndexXaot >= argIndexXnoaot) {
+						vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_AOT;
+#if defined(J9VM_ARCH_X86)
+						if (argIndexXXPortableSharedCache > argIndexXXNoPortableSharedCache) {
+							vm->extendedRuntimeFlags2 |= J9_EXTENDED_RUNTIME2_ENABLE_PORTABLE_SHARED_CACHE;
+						} else if (argIndexXXPortableSharedCache == argIndexXXNoPortableSharedCache) {
```

Comment:
Does this `==` case means that these options are not found in the command line ?

Reply:
Yes, == means not found so we follow the default behavior which is enabled by default in containers and disabled by default otherwise.

Reply:
OK. Then please add a comment saying that none of these options are used in this case.